### PR TITLE
Testcase for merge operator

### PR DESCRIPTION
**Pull request by ritesh-singh**

### CODE REVIEW

**Comment by sergiiz** [comment]
Thanks for your PRs @ritesh-singh, I'll take a look later this week

File: rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java [hunk 1]
```diff
@@ -31,4 +31,12 @@
 
     Single<Map<String, Long>> mapCountriesToNamePopulation(List<Country> countries);
 
+    Observable<Long> combineAndReturnTheSumOfPopulation(Observable<Country> countryObservable1,
```
Comment (sergiiz) [inline rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java]: Rename: sumPopulationOfCountries

File: rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java [hunk 2]
```diff
@@ -31,4 +31,12 @@
...
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfBothObservableEmitsSameItems(Observable<Country> countryObservable1,
```
Comment (sergiiz) [inline rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java]: Rename: areEmittingSameItems (better names are welcome)

File: rxkata/src/test/java/org/sergiiz/rxkata/CountriesServiceSolvedTest.java [hunk 3]
```diff
@@ -164,4 +165,56 @@ public void rx_MapCountriesToNamePopulation() {
         values.assertResult(expected);
         values.assertNoErrors();
     }
+
+    @Test
+    public void rx_CombineTwoObservablesAndReturnSumOfPopulation(){
+        // hint: use "map" operator
```
Comment (sergiiz) [inline rxkata/src/test/java/org/sergiiz/rxkata/CountriesServiceSolvedTest.java]: Please rename all tests according to the following pattern: rx_<Tested_Method_Name> with behaviour explanation divided by `_`, see other test cases names as example

File: README.md [hunk 4]
```diff
@@ -20,6 +20,8 @@ A set of simple code challenges to learn RxJava using JUnit tests as an acceptan
 - all: determines whether all items meet some criteria
 - reduce: applies a function to each item sequentially, and emit the final value. For example, it can be used to sum up all emitted items
 - toMap: converts an Observable into another object or data structure
+- merge: combine multiple Observables into one by merging their emissions
+- SequenceEqual: determine whether two Observable emit the same sequence of items
```
Comment (sergiiz) [inline README.md]: typo Observable**s**

File: rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java [hunk 5]
```diff
@@ -31,4 +31,12 @@
...
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfObservableEmitsGivenItem(List<Country> countries,
```
Comment (sergiiz) [inline rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java]: Rename: ifObservableEmitsGivenItem.
And I'm wondering if this case is important at all, have you faced real situations where you needed such check?

**Review by sergiiz — CHANGES_REQUESTED** [review]

**Comment by ritesh-singh** [comment]
Sure. I will do the required changes and push.

File: rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java (hunk 5 above)
Comment (ritesh-singh) [reply rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java]: Actually no, i have added it just to test the 'contains' operator functionality.

**Pull request merged**